app/controllers/assessments_controller.rb

```diff
@@ -219,6 +232,7 @@ def importAsmtFromTar
     end
 
     params[:assessment_name] = asmt_name
+    params[:isTarImport] = true
```

Comment:
Perhaps it would be more meaningful to call the param `cleanup_on_failure`? i.e. describe the behavior, rather than the scenario